setup.py

```diff
@@ -52,7 +50,9 @@
     'develop', 'release', 'bdist_egg', 'bdist_rpm',
     'bdist_wininst', 'install_egg_info', 'build_sphinx',
     'egg_info', 'easy_install', 'upload', 'bdist_wheel',
-    '--single-version-externally-managed',
+    '--single-version-externally-managed', 'install', 'build', 'build_ext',
```

Comment:
Actually, never mind the last bit. You need ``setuptools`` for ``develop``. The strategy of avoiding ``setuptools`` whenever possible is sensible, but it places higer requirements on CI coverage to avoid only running into bugs like this at release time. Because the majority of people install via ``pip`` nowadays, and then you get ``setuptools`` for free anyway:(

I'll remove the first commit.